internal/admission/validator.go

```diff
@@ -169,20 +172,40 @@ func (validator KongHTTPValidator) ValidateConsumerGroup(
 		return true, "", nil
 	}
 
-	cgs, ok := validator.AdminAPIServicesProvider.GetConsumerGroupsService()
+	cgs, ok := validator.AdminAPIServicesProvider.GetConsumerGroupService()
 	// If there's no client, do not verify with data-plane as there's none available.
 	if !ok {
 		return true, "", nil
 	}
-	fmt.Println(">>>>>>>>>> WEBHOOK")
-	// check version too
+	info, ok := validator.AdminAPIServicesProvider.GetInfoService()
+	if !ok {
+		return true, "", nil
+	}
+
+	infoResponse, err := info.Get(ctx)
+	if err != nil {
+		return false, ErrTextAdminAPIUnavailable, err
+	}
+	version, err := kong.NewVersion(infoResponse.Version)
+	if err != nil {
+		// if we can't parse the version, assume weird custom build try anyway
+		return true, "", nil
```

Comment:
It doesn't seem right to me to silently accept any invalid semver here with no further checks. I think we should rather skip the semantic checks (`IsKongGatewayEnterprise` and the cutoff version) but proceed with the `cgs.List` heuristic that will give us an answer to whether ConsumerGroups are supported or not despite the version we get.

Allowed myself to apply these changes straight away, feel free to drop them if you find them invalid: 
2a0fd17